test/executor.test.js

```diff
@@ -434,9 +435,11 @@ describe('executor', function() {
       .get('/')
       .end(function(err, res) {
         if (err) return done(err);
-        expect(res.text).to.eql('<!DOCTYPE html>\n<html>\n<head lang="en">\n' +
-          '    <meta charset="UTF-8">\n    <title>simple-app</title>\n' +
-          '</head>\n<body>\n<h1>simple-app</h1>\n</body>\n</html>');
+        expect(res.text).to.eql('<!DOCTYPE html>' + os.EOL + '<html>' + os.EOL +
+          '<head lang="en">' + os.EOL + '' + '    <meta charset="UTF-8">' +
+          os.EOL + '    <title>simple-app</title>' + os.EOL + '</head>' +
+          os.EOL + '<body>' + os.EOL + '<h1>simple-app</h1>' + os.EOL +
+          '</body>' + os.EOL + '</html>');
```

Comment:
I'm wondering if there is any cleaner way how to fix this, so that the text can keep using `\n` characters. How about something like this?

```js
expect(res.text).to.eql('....'.replace(/\n/g, os.EOL));
```